lib/drush-node.js

```diff
@@ -1,5 +1,7 @@
+/* globals require, module, console */
```

Comment:
Hm, looks like the node option was already set. Might have been a hiccup in my syntastic config because I was seeing errors in vim and now I'm not. I'll remove it.